README.md

```diff
@@ -1,15 +1,101 @@
 # getappmap/archive-appmap
 
+> To get started with AppMap in GitHub actions, you need to start by installing the [AppMap App on the official GitHub Marketplace](https://github.com/marketplace/get-appmap)
+
+> To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+
 GitHub action to build and store an AppMap archive.
 
-To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Inputs](#inputs)
+- [Examples](#examples)
+- [Development](#development)
+
+## Prerequisites
```

Comment:
In the install-action I changed this to Requirements - https://github.com/getappmap/install-action#requirements - which seemed more clear to me